gadget/gadget_test.go

```diff
@@ -639,13 +639,16 @@ func mustParseStructure(c *C, s string) *gadget.VolumeStructure {
 func (s *gadgetYamlTestSuite) TestValidateRole(c *C) {
 	uuidType := `
 type: 21686148-6449-6E6F-744E-656564454649
+size: 1023
 `
 	bareType := `
 type: mbr
+size: 446
```

Comment:
It could be interesting to also test mbr with a size of 447, to make sure our limit works.